lib/availability_resolver.js

```diff
@@ -26,9 +26,10 @@ class AvailabilityResolver {
         // If this serialization is a result of a hold request initializing, we want
         // to double check the recap customer code in SCSB
         if (options && options.queryRecapCustomerCode) {
-          this._checkScsbForRecapCustomerCode()
-        }
-        return this.elasticSearchResponse
+          this._checkScsbForRecapCustomerCode().then(() => {
```

Comment:
Gotta return this, or else it's just kicking off `this._checkScsbForRecapCustomerCode()` and continuing